packages/strapi/lib/middlewares/responses/index.js

```diff
@@ -22,8 +22,8 @@ module.exports = () => {
         }
 
         // Set X-Powered-By header.
-        if (_.get(strapi.config, 'X-Powered-By.enabled', true)) {
-          ctx.set('X-Powered-By', 'Strapi <strapi.io>');
+        if (_.get(strapi.config.currentEnvironment.response, 'poweredBy.enabled', null)) {
+          ctx.set('X-Powered-By', _.get(strapi.config.currentEnvironment.response, 'poweredBy.value'));
```

Comment:
Can you set `Strapi <strapi.io>` as default value of the `_.get` function please.